api/v1beta1/placementapi_webhook.go

```diff
@@ -86,7 +92,25 @@ func (r *PlacementAPI) ValidateCreate() error {
 func (r *PlacementAPI) ValidateUpdate(old runtime.Object) error {
 	placementapilog.Info("validate update", "name", r.Name)
 
-	// TODO(user): fill in your validation logic upon object update.
+	oldPlacement, ok := old.(*PlacementAPI)
+	if !ok || oldPlacement == nil {
+		return apierrors.NewInternalError(fmt.Errorf("unable to convert existing object"))
+	}
+
+	if ! r.Spec.AdvancedMode {
```

Comment:
do we want an explicite extra step to toggle the `advancedMode` ? then we should check this in the old object. if we are ok that updating `advancedMode` and other parameters in one go this is fine.